Lushu/build.gradle.kts

```diff
@@ -2,7 +2,7 @@ import org.gradle.api.tasks.testing.logging.TestExceptionFormat
 import org.gradle.api.tasks.testing.logging.TestLogEvent
 
 repositories {
-    jcenter()
+    mavenCentral()
```

Comment:
Why are we changing to maven? I prefer sticking to jcenter unless strictly necessary to make the change.